_data/meltano/extractors/tap-partnerize/voxmedia.yml

```diff
@@ -0,0 +1,39 @@
+capabilities:
+- catalog
+- discover
+- state
+- about
+- stream-maps
+description: Partnerize API
+domain_url: https://api-docs.partnerize.com/partner
+keywords:
+- api
+- meltano_sdk
+- partnerize
```

Comment:
```suggestion
```

@omarabed15 these keywords are mostly for helping with search so you shouldnt need `partnerize` since its part of the name/label already.

Reply:
Good call, done! 👍